CRMImporter/Converters/StringMapTranslation.cs

```diff
@@ -0,0 +1,27 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.Xrm.Sdk;
+
+namespace CRMImporter.Converters
+{
+    public class StringMapTranslation : IConverter
+    {
+        private readonly Dictionary<string, string> mapping;
+        public StringMapTranslation(Dictionary<string, string> mapping)
+        {
+            this.mapping = mapping;
+        }
+        public object Convert(object input, IOrganizationService service)

```

Comment:
This function doesn't seem to use `service`.  Is it needed?

Reply:
No it isn't needed but it needs it to satisfy the interface